src/CompleteBipartiteGraph.jl

```diff
@@ -0,0 +1,158 @@
+
+
+# =======================================================
+#         CompleteBipartiteGraph struct
+# =======================================================
+
+"""
+    CompleteBipartiteGraph <: AbstractGraph
+
+A structure representing an undirected bipartite graph.
+
+
+See also: [`LightGraphs.complete_bipartite_graph`](@ref)
+"""
+struct CompleteBipartiteGraph{T<:Integer} <: LG.AbstractGraph{T}
+    m::T
+    n::T
+
+    function CompleteBipartiteGraph{T}(m, n) where {T}
+       
+        m = convert(T, m)
+        n = convert(T, n)
+        m >= zero(T) || throw(ArgumentError("m must be >= 0"))
+        n >= zero(T) || throw(ArgumentError("n must be >= 0"))
+        _, does_overflow = Base.Checked.add_with_overflow(m, n)
+        does_overflow && throw(ArgumentError("m + n must not exceed typemax($T)"))
+
+        return new{T}(m, n)
+    end
+end
+
+CompleteBipartiteGraph(m::T, n::T) where {T<: Integer} = CompleteBipartiteGraph{T}(m, n)
+
+
+# =======================================================
+#         traits
+# =======================================================
+
+LG.is_directed(::Type{<:CompleteBipartiteGraph}) = false
+
+
+# =======================================================
+#        vertices
+# =======================================================
+
+Base.eltype(::Type{CompleteBipartiteGraph{T}}) where {T} = T
+Base.eltype(g::CompleteBipartiteGraph) = eltype(typeof(g))
+
+LG.nv(g::CompleteBipartiteGraph{T}) where {T} = T(g.m + g.n)
```

Comment:
is the conversion to `T` needed here? Addition of two `T` should preserve type

Reply:
Right, I removed that now.